proxy/src/avatar.rs

```diff
@@ -32,6 +61,27 @@ impl fmt::Display for Emoji {
     }
 }
 
+/// Avatar usage.
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub enum Usage {
+    /// A user avatar.
+    User,
+    /// An org avatar.
+    Org,
+}
+
+impl std::str::FromStr for Usage {
```

Comment:
I'd drop this and make the it the callers responsibility to use the appropriate enum variant.